src/main/java/kitEditor/KitEditor.java

```diff
@@ -1188,9 +1189,9 @@ private void duplicateSample(Sample sample) {
     }
     
     private void pasteSample() {
-        for (Sample sample : clipboard) {
-            if (sample != null) {
-                duplicateSample(sample);
+        for (int index = 0; index < clipboard.length; ++index) {
+            if (clipboard[index] != null) {
+                duplicateSample(clipboard[index]);
```

Comment:
what is the reason for this change?
was there some kind of merge problem?

Reply:
yes, this change is so that multiple samples can be copied and pasted, rather than only one